src/shared/clients/ecsClient.ts

```
@@ -77,6 +75,39 @@ export class DefaultEcsClient {
         }
     }
 
+    public async listTasks(cluster: string, serviceName: string): Promise<string[]> {
+        const sdkClient = await this.createSdkClient()
+
+        const params: ECS.ListTasksRequest = { cluster: cluster, serviceName: serviceName }
+        try {
+            const listTasksResponse = await sdkClient.listTasks(params).promise()
+            return listTasksResponse.taskArns ?? []
+        } catch (error) {
+            getLogger().error(
```

Just let the errors bubble up and handle them further up the stack.